Hearthstone/app/src/main/java/com/example/hearthstone/data/network/repo/HSRepo.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.hearthstone.data.network.repo
+
+import com.example.hearthstone.data.model.HearthstoneModel
+import retrofit2.Response
+
+interface HSRepo {
+    suspend fun getCards(): HearthstoneModel?
+    companion object{
+        fun provideHSRepoApi() = HSRepoImpl()
```

Comment:
are you going to do DI for this?

Reply:
I applied it in another branch, in this one it still didn't